src/main/resources/templates/style.css

```diff
@@ -1,2 +1,7 @@
-body{font-family: Consolas,serif;}
-.panel{ width: 400px; height: 400px; border: solid black 1px}
\ No newline at end of file
+body{
+    font-family: Consolas,serif;
+}
+
+.panel{
+    width: 400px; height: 400px; border: solid black 1px
+}
```

Comment:
Not a very substantial commit, but I fixed this to abide by what most would consider "industry standard" for CSS formatting. Single line CSS is hard to read and frustrating to edit.

https://css-tricks.com/different-ways-to-format-css/

There are other ways you can format your stylesheet as it grows larger. Refer to the document above for some recommendations.